packages/hop-node/src/theGraph/getTransferFromL1Completed.ts

```diff
@@ -0,0 +1,58 @@
+import makeRequest from './makeRequest'
+import { MaxInt32 } from 'src/constants'
+import { constants } from 'ethers'
+import { normalizeEntity } from './shared'
+
+export default async function getTransferFromL1Completed (
+  chain: string,
+  token: string,
+  startDate: number = 0,
+  endDate: number = MaxInt32,
+  lastId: string = constants.AddressZero
```

Comment:
I know we use `constants.AddressZero` in multiple places as default `lastId` but they can all be simplified by just using `"0"`

```ts
lastId: string = '0'
```

since technically the `id` is not an address and it just needs to be a low string value